tests/network-utils/test_network_utils.py

```diff
@@ -14,59 +14,84 @@
     ("chain_id", "expected_name"),
     [
         (1, "Ethereum Mainnet"),
-        (249, None),
         (2, "Expanse Network"),
         (42, "LUKSO Mainnet"),
-        (214, None),
     ],
 )
 def test_network_name_from_chain_id(chain_id, expected_name):
-    if expected_name is not None:
-        result = name_from_chain_id(chain_id)
-        assert result == expected_name
-    else:
-        with pytest.raises(ValidationError) as ex:
-            name_from_chain_id(chain_id)
-        assert "chain_id is not recognized" in str(ex.value)
+    result = name_from_chain_id(chain_id)
+    assert result == expected_name
+
+
+# Use large chainId values that should never exist
+@pytest.mark.parametrize(
```

Comment:
All of these could be simplified:

- They all raise `ValidationError`, so we shouldn't have to parametrize it
- We can use the `chain_id` value in the `match=` kwarg of `pytest.raises()` as such:

```py
@pytest.mark.parametrize("chain_id", (222**222, 333**333, 444**444))
def test_network_name_from_chain_id_with_invalid_chain_id(chain_id):
    with pytest.raises(
        ValidationError, match=f"chain_id is not recognized: {chain_id}"
    ):
        network_from_chain_id(chain_id)
```

It would be nice to DRY all three tests into one as well:

```py
@pytest.mark.parametrize("chain_id", (222**222, 333**333, 444**444))
def test_from_chain_id_utility_methods_with_invalid_chain_id(chain_id):
    error_message = f"chain_id is not recognized: {chain_id}"

    # test network_from_chain_id
    with pytest.raises(ValidationError, match=error_message):
        network_from_chain_id(chain_id)

    # test short_name_from_chain_id
    with pytest.raises(ValidationError, match=error_message):
        short_name_from_chain_id(chain_id)

    # test name_from_chain_id
    with pytest.raises(ValidationError, match=error_message):
        name_from_chain_id(chain_id)
```